### PR TITLE
fixed a couple pages which refer to services instead of applications

### LINKED ISSUE #1645: lots of things called services which should now be applications.
https://jujucharms.com/docs/stable/authors-hook-environment

### PR DESCRIPTION
**Pull request by matthewhelmke**
Fixes #1645 
This is a short term fix for a bigger issue. The charms docs are outdated. We know this and are working toward a massive overhaul. In the meanwhile, this fixes references on a few pages. Much more work is needed and forthcoming.

### CODE REVIEW

File: src/en/authors-charm-leadership.md [hunk 1]
```diff
@@ -48,14 +48,14 @@ unprepossessing race scenarios.
...
-leader settings can be communicated with the following tools:
+Every application deployed by juju also has access to a pseudo-relation over
+which leader settings can be communicated with the following tools:
 
   * `leader-set` acts much like `relation-set`, in that it lets you write string
     key/value pairs (in which an empty value removes the key), but with the
     following differences:
 
-      * there's only one leader-settings bucket per service (not one per unit)
+      * there's only one leader-settings bucket per applicaiton (not one per unit)
```
Comment (evilnick) [inline src/en/authors-charm-leadership.md]: application

File: src/en/authors-charm-leadership.md [hunk 2]
```diff
@@ -48,14 +48,14 @@ unprepossessing race scenarios.
     to verify continued leadership past lease expiry time, it would start to
     return false.
 
-Every service deployed by juju also has access to a pseudo-relation over which
-leader settings can be communicated with the following tools:
+Every application deployed by juju also has access to a pseudo-relation over
```
Comment (evilnick) [inline src/en/authors-charm-leadership.md]: Juju

File: src/en/authors-charm-leadership.md [hunk 3]
```diff
@@ -2,12 +2,12 @@ Title: Implementing leadership in Juju charms
...
 
-Leadership provides a mechanism whereby multiple units of a service can make
-use of a single, shared, authoritative source for charm-driven configuration
+Leadership provides a mechanism whereby multiple units of an application can
+make use of a single, shared, authoritative source for charm-driven configuration
 settings.
 
-Every service deployed by juju is guaranteed to have at most one leader at any
-time. This is true independent of what the charm author does; whether or not
+Every application deployed by juju is guaranteed to have at most one leader at
```
Comment (evilnick) [inline src/en/authors-charm-leadership.md]: Juju

Comment (matthewhelmke) [inline src/en/authors-charm-leadership.md]: Doh!

File: src/en/authors-charm-leadership.md (hunk 2 above)
Comment (matthewhelmke) [inline src/en/authors-charm-leadership.md]: smh

Comment (matthewhelmke) [inline src/en/authors-charm-leadership.md]: Found several more on the page, too...

**Pull request merged**